### PR TITLE
feat: Fix oracle compilation (SC-3061)

### PR DESCRIPTION
**Pull request by satyamakgec**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+do
+    case "${flag}" in
+        c) config=${OPTARG};;
+    esac
+done
+
+config=$([ -z "$config" ] && echo "./config/prod.json" || echo "$config")
+
+# Workaround for the src folder
+mv -f lib/ds-test/src lib/ds-test/contracts
```
Comment (lucas-manuel) [inline build.sh]: Shouldn't be needed

File: build.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+export DAPP_TEST_NUMBER=12543537
+export DAPP_SOLC_VERSION=0.6.11
+export DAPP_SRC="contracts"
+export DAPP_LINK_TEST_LIBRARIES=0
+export DAPP_STANDARD_JSON=$config
+
+dapp --use solc:0.6.11 build
+
+# Workaround for the src folder
+mv -f lib/ds-test/contracts lib/ds-test/src
```
Comment (lucas-manuel) [inline build.sh]: Shouldn't be needed

File: build.sh (hunk 1 above)
Comment (lucas-manuel) [reply build.sh]: Need to move to `modules`

**Review by lucas-manuel — CHANGES_REQUESTED** [review]

**Comment by lucas-manuel** [comment]
We also need to update CI config file to run tests, you can refer to this [file](https://github.com/maple-labs/late-fee-calculator/blob/dd5e4ddc3fd7f1bc5eb72344bf1644af06aa1e51/.circleci/config.yml).

**Review by deluca-mike — DISMISSED** [review]

File: .circleci/config.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 version: 2.1
 
 jobs:
-  dapp_build:
+  dapp_test:
```
Comment (lucas-manuel) [inline .circleci/config.yml:4]: We should do a build at least I think

**Review by lucas-manuel — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 3 above)
Comment (deluca-mike) [reply .circleci/config.yml:4]: Disagree. CI tests (even if there are no tests) and thus builds (which will act as a check that the code is at least valid as well), so if tests are ever added, they are automatically run. CD will eventually build for the purposes of deployment/packaging.

**Pull request merged**